src/App.css

```diff
@@ -28,6 +28,49 @@
   color: #61dafb;
 }
 
+.my-task {
+  padding-left: 50px;
+  width: 250 px;
+  height: 50px;
+  border: solid;
+  display: flex;
+  justify-content: center;
+}
```

Comment:
I don't think your UI code uses this `className` at all. Can you double-check and remove if that's true?